rector.php

```diff
@@ -12,6 +12,12 @@
 use Rector\Set\ValueObject\LevelSetList;
 
 return static function ( RectorConfig $rector_config ): void {
+	$rector_config->bootstrapFiles(
```

Comment:
For some reason, Rector stopped seeing WordPress classes. This piece of code makes this work again.